packages/common/driver-definitions/src/storage.ts

```diff
@@ -450,3 +461,43 @@ export enum FetchSource {
 	default = "default",
 	noCache = "noCache",
 }
+
+/**
+ * @alpha
+ */
+export interface ISnapshot {
+	snapshotTree: ISnapshotTree;
+	blobContents: Map<string, ArrayBuffer>;
+	ops: ISequencedDocumentMessage[];
+
+	/**
+	 * Sequence number of the snapshot
+	 */
+	sequenceNumber: number | undefined;
+
+	/**
+	 * Sequence number for the latest op/snapshot for the file in ODSP
+	 */
+	latestSequenceNumber: number | undefined;
+
+	snapshotInNewFormat: true;
+}
+
+/**
+ * Additional key in the loader request header
+ * @alpha
+ */
+export enum GetSnapshotOptions {
```

Comment:
I'm not sure why we need this enum. It increases bundle size.
I do not see any place where it's used out then in defining ISnapshotFetchOptions interface, and that can be done without enum

Reply:
Removed.